src/slides/class3/slides/Modules.jsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import {
+  Slide,
+  Heading,
+  Text,
+  UnorderedList,
+  ListItem,
+  CodePane,
+  Notes,
+} from "spectacle";
+
+export default () => (
+  <>
+    <Slide backgroundColor="#0d0d0d">
+      <Heading margin="0px" fontSize="50px">
+        Módulos
+      </Heading>
+
+      <Text fontSize={30}>
+        Los programas escritos en JavaScript comenzaron siendo bastante pequeños
+        (tareas de scripting aisladas o adición de interactividad a páginas web)
+        por lo que generalmente no se necesitaban grandes scripts.
+      </Text>
+
+      <Text fontSize={30}>
+        El uso de JavaScript ha ido evolucionando, y ahora tenemos aplicaciones
+        completas que se ejecutan en navegadores utilizando librerías como
+        ReactJS o frameworks como Angular. Además, JS ahora se usa en otros
+        contextos (Node.js por ejemplo).
+      </Text>
+
+      <Text fontSize={30}>
+        En los últimos años se ha comenzado a pensar en proporcionar mecanismos
+        para dividir programas JavaScript en módulos separados que se puedan
+        importar cuando sea necesario. Hay una serie de librerías y sistemas que
+        permiten el uso de módulos, siendo una de las más adoptadas{" "}
+        <strong>CommonJS.</strong>
+      </Text>
+    </Slide>
+
+    <Slide backgroundColor="#0d0d0d" style={{ display: "flex" }}>
+      <Heading margin="0px" fontSize="50px">
+        CommonJS
+      </Heading>
+
+      <Text fontSize={30}>
+        Es el sistema de módulos utilizado por <strong>Node.js</strong> y consta
```

Comment:
You can change the config, so...
```suggestion
        Es el sistema de módulos utilizado por defecto por <strong>Node.js</strong> y consta
```

Reply:
That sounds better!